samples/bluetooth/ipsp/src/main.c

```diff
@@ -38,7 +38,9 @@ struct in6_addr in6addr_mcast = MCAST_IP6ADDR;
 
 static struct in6_addr in6addr_my = MY_IP6ADDR;
 
-#define MY_PORT 4242
+#ifndef CONFIG_NET_CONFIG_MY_PORT
+#define CONFIG_NET_CONFIG_MY_PORT 4242
+#endif
 
```

Comment:
Please do this other way around like this
```
#if defined(CONFIG_NET_CONFIG_MY_PORT)
#define MY_PORT CONFIG_NET_CONFIG_MY_PORT
#else 
#define MY_PORT 4242
#endif 
```
Then we only need to change this initial setting of MY_PORT